airtable/airtable.py

```diff
@@ -435,6 +435,26 @@ def update(self, record_id, fields, typecast=False):
             record_url, json_data={"fields": fields, "typecast": typecast}
         )
 
+    def batch_update(self, records, typecast=False):
+        """
+        Updates a records by their record id's in batch.
+
+        Args:
+            records(``dict``): List of dict: [{"id": record_id, "field": fields_to_update_dict}]

```

Comment:
Could you add missing `Arg` documentation?
https://github.com/gtalarico/airtable-python-wrapper/blob/d829743342c34e11e1fb38fc5579a0533af4900a/airtable/airtable.py#L524